packages/binding-modbus/src/modbus-client.ts

```diff
@@ -8,6 +8,7 @@ import { SecurityScheme } from '@node-wot/td-tools'
 import { modbusFunctionToEntity } from './utils'
 import { ModbusConnection, PropertyOperation } from './modbus-connection'
 import { Readable } from 'stream'
+import { Subscription } from "rxjs/Subscription"
```

Comment:
Shouldn't be already provided by the core module? how did it work previously without explicit dependency?

Reply:
There was no usage of Subscription class inside modbus package previously so I just proceeded in the same was as other packages, for example https://github.com/relu91/thingweb.node-wot/blob/ceab936ff65e82b15498c024e7d2b923150c6c6f/packages/binding-coap/src/coap-client.ts#L23